weighted_graph/weight_graph.py

```diff
@@ -0,0 +1,94 @@
+# _*_ coding: utf-8 _*_
+"""Graph Data Structure."""
+
+
+class Graph(object):
+    """Class of the Graph Data Structure."""
```

Comment:
You've named your simple graph and your weighted graph "Graph". You shouldn't have overlapping names in the same package.